magento/pwa-studio
# [Theming] Base style buttons

## Description

Convert the Button and ButtonGroup component to fit the Base design for Buttons using Tailwind.

Address accessibility issues on various buttons throughout Venia.

## Related Issue







Closes PWA-2657

## Acceptance





Any developer on the DDG team

@schensley 

### Verification Stakeholders



Any developer on the DDG team

@schensley 

### Specification




## Verification Steps






1. Run storybook: `yarn storybook:venia`
2. Open Buttons and ButtonGroup stories
3. Verify stories visually match base design

#### Test scenario(s) for direct fix/feature

1. Normal state
2. Hover state
3. Focus state





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?

Probably not. To keep this PR small, I did not try to fix minor issues on pages that use Buttons and instead will rely on future base conversion work on those components/pages to address them.





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)

Button and ButtonGroup prop signature have been changed.





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
